cmd/krew/cmd/update.go

```diff
@@ -129,6 +129,7 @@ func ensureIndexUpdated(_ *cobra.Command, _ []string) error {
 		installedPlugins[receipt.Name] = receipt.Spec.Version
 	}
 
+	// TODO(chriskim06) consider commenting this out when refactoring for custom indexes
```

Comment:
not quite commenting out, but I think we need to pass a similar Pair<indexName,plugin> here as well :) :)

Reply:
Ya `pluginEntry` would be available here. I was just making a note of possibly commenting out the show updated plugins feature like you mentioned in your other comment